lib/kittikrb/cursor.rb

```diff
@@ -115,6 +146,113 @@ def reset_tty!
         self
       end
 
+      def erase(x1, y1, x2, y2)
+        y2.floor.downto(y1.floor) do |y|
+          x1.floor.upto(x2.floor) do |x|
+            move_to x, y
+            write_control "\s"
+          end
+        end
+
+        self
+      end
+
+      def eraseToEnd
```

Comment:
I know, it is my fault, sorry. But in ruby, all methods should be named in snake_case, not in camelCase. Please, fix it everywhere.